cmd/vpu_plugin/vpu_plugin.go

```diff
@@ -55,18 +57,28 @@ type devicePlugin struct {
 	vendorID     int
 	productIDs   []int
 	sharedDevNum int
+	scanTicker *time.Ticker
+	scanDone chan bool
 }
 
 func newDevicePlugin(usbContext gousbContext, vendorID int, productIDs []int, sharedDevNum int) *devicePlugin {
-	return &devicePlugin{
+	if sharedDevNum < 1 {
+		klog.V(1).Info("The number of containers sharing the same VPU must greater than zero, force it to 1")
+		sharedDevNum = 1
+	}
+	dp := devicePlugin{
 		usbContext:   usbContext,
 		vendorID:     vendorID,
 		productIDs:   productIDs,
 		sharedDevNum: sharedDevNum,
 	}
+	dp.scanTicker = time.NewTicker(scanFrequency)
+	dp.scanDone = make(chan bool, 1)
```

Comment:
These two assignments should be moved to `dp` initialization:
```golang
	dp := devicePlugin{
		usbContext:   usbContext,
		vendorID:     vendorID,
		productIDs:   productIDs,
		sharedDevNum: sharedDevNum,
                scanTicker: time.NewTicker(scanFrequency),
                scanDone: make(chan bool, 1),
	}
```

Reply:
done. thanks!